doc/doc.md

```diff
@@ -1203,7 +1203,7 @@ _These only appear in the configuration file:_
   - `full_description` when you _really_ need a longer project description
   - `examples` a directory or file: can be a table
   - `readme` or `topics` readme files (to be processed with Markdown)
-  - `pretty` code prettify 'lua' (default) or 'lxsh'
+  - `pretty` code prettify 'lua' (default), 'lxsh' or 'none'
```

Comment:
Why shouldn't this be `false`? We can fix up the issue of testing for booleans if it was a matter of not being able to figure that out. Is there another reason 'none' is preferred?

Reply:
I take this pack, 'none' is consistent with other option usage here, so we'll go with it.